.vscode/tasks.json

```diff
@@ -16,21 +16,6 @@
 			"args": [ "dotnet restore --no-cache" ]
 		},
 		{
```

Comment:
Shouldn't we fix the command line instead of removing it?

Reply:
I thought these are for `project.json` and we don't need them and I think down below has the same tasks. No?

Reply:
You are right, I had not seen the rest of the file